packages/xml-views-quick-fix/test/quick-fix-stable-id-spec.ts

```diff
@@ -5,6 +5,8 @@ import { buildAst, XMLDocument } from "@xml-tools/ast";
 import { expectExists } from "@ui5-language-assistant/test-utils";
 import { computeQuickFixStableIdInfo } from "../src/quick-fix-stable-id";
 
+const UNFOUND_CHAR = -1;
```

Comment:
indexOf !== -1 is idiomatic in JavaScript.
I actually found: `const UNFOUND_CHAR = -1;` to be confusing. (pun intended).

That is because:
- you probably meant: "**NOT_FOUND**_CHAR" because `unfounded` means something else:
  - https://www.dictionary.com/browse/unfounded
- The `_CHAR` is likely redundant because:
  - We don't care what we are searching for or on what object we are searching on, e.g:
    - Lodash's `findIndex` also returns `-1` when **NOT_FOUND**.
 
Options:
1. Use const.
  - call it `NOT_FOUND`
  - define it directly above the **single** function it is used in.
2. Use `-1` directly.